cosmic-app-list/i18n/pl/cosmic_app_list.ftl

```diff
@@ -1,5 +1,5 @@
 cosmic-app-list = Lista Apek Doku Cosmic
```

Comment:
On a second thought, there is no "Cosmic" in the English version, so maybe drop that too.

Reply:
Well, formerly it was "Cosmic Dock App List", now changed to "App tray" which translate directly to Zasobnik Apek or more formally "Zasobnik Aplikacji". There is also a case for "Tacka Apek/Aplikacji", but imo it's too focused on physical attributes of tray, not on metaphorical use case.

Anyway, need to be consistent across the board.

Reply:
I've looked around and some phones use "Zasobnik Aplikacji" to describe their pull-down app menus. This isn't exactly that, but it could be used. So then line 1 should be updated to that if line 2 mentions "Zasobnik".